style.css

```diff
@@ -1,8 +1,167 @@
 #app {
```

Comment:
Any particular reason to leave this selector and styles here?

Reply:
The content is wrapper inside app div.